core/build.gradle.kts

```diff
@@ -71,49 +79,32 @@ kotlin {
     }
 
     sourceSets {
-        commonMain {
-            dependencies {
-                api("org.jetbrains.kotlin:kotlin-stdlib-common")
-            }
+        val commonMain by getting {
         }
-
-        commonTest {
+        val commonTest by getting {
             dependencies {
-                api("org.jetbrains.kotlin:kotlin-test-common")
-                api("org.jetbrains.kotlin:kotlin-test-annotations-common")
+                implementation(kotlin("test"))
             }
         }
 
         val jvmMain by getting {
-            dependencies {
-                api("org.jetbrains.kotlin:kotlin-stdlib")
-
-            }
         }
         val jvmTest by getting {
             dependencies {
-                api("org.jetbrains.kotlin:kotlin-test-junit")
                 implementation("com.google.guava:guava-testlib:18.0")
             }
         }
 
         val jsMain by getting {
-            dependencies {
-                api("org.jetbrains.kotlin:kotlin-stdlib-js")
-            }
         }
-
         val jsTest by getting {
-            dependencies {
-                api("org.jetbrains.kotlin:kotlin-test-js")
-            }
         }
 
         val nativeMain by getting {
-            dependsOn(commonMain.get())
+            dependsOn(commonMain)
```

Comment:
Optional: you can update infra plugin to 0.4.0-dev-80 to get rid of this explicit dependsOn.

Reply:
Done